orchagent/portsorch.cpp

```diff
@@ -1772,6 +1839,15 @@ bool PortsOrch::addLagMember(Port &lag, Port &port)
 
     m_portList[lag.m_alias] = lag;
 
+    if (lag.m_bridge_port_id > 0)
```

Comment:
also check if the port belongs to multiple vlans or not, if yes then add keep, otherwise do not need to do anything.